plugins/Installation/Controller.php

```diff
@@ -149,6 +149,9 @@ function databaseSetup()
 
                 DbHelper::checkDatabaseVersion();
 
+                // check if database is UTF8MB4 
+                $dbInfos['enable_dynamic_row_format'] = Db::get()->isConnectionUTF8MB4() ? 1 : 0;
```

Comment:
fyi I believe here we can simplify this using 
```php
        $config->database['enable_dynamic_row_format'] = $config->database['charset'] === 'utf8mb4' ? 1 : 0;
```

in the `createConfigFile` method. Then you won't need any of the methods using `isConnectionUTF8MB4` I suppose

Reply:
@tsteur that's right, let me tide this up.